Submitted By:            Jeff Bai <jeffbaichina@gmail.com>
Date:                    2013-07-18
Initial Package Version: 4.10.5
Description:             Fixes ugly icon spacing in systemsettings

diff -Naur a/systemsettings/categories/settings-locale.desktop b/systemsettings/categories/settings-l$
--- a/systemsettings/categories/settings-locale.desktop 2013-06-29 01:10:44.759399536 +0800
+++ b/systemsettings/categories/settings-locale.desktop 2013-07-18 15:17:04.087168760 +0800
@@ -68,5 +68,5 @@
 Name[vi]=Vùng
 Name[wa]=Lingaedje/payis
 Name[x-test]=xxLocalexx
-Name[zh_CN]=语系
-Name[zh_TW]=地域
+Name[zh_CN]=语系 (Localization)
+Name[zh_TW]=地域 (Localization)
